os-module.js

```diff
@@ -0,0 +1,9 @@
+import { platform, arch, cpus, totalmem, freemem } from 'os';
+
+export const getSystemInfo = () => ({
+  operatingSystem: platform(),
+  cpuArchitecture: arch(),
+  numberOfCPUs: cpus().length,
+  totalMemory: (totalmem() / (1024 ** 3)).toFixed(2),
```

Comment:
I would suggest to create a separate method with meaningful name to convert to GB